### DISCUSSION TITLE
How to setup a connection to a chain without gRPC enabled

### DISCUSSION THREAD

**Discussion by simpletrontdip** [informalsystems/hermes#1949] [Q&A] [answered]
## Create
relayer

## Problem
This is a question, not really a bug.
- `grpc_addr` is required in chain config
- Some cosmos chain does not enable gRPC in their node, e.g. **Akash Testnet-3**

How can I set up Hermes to connect to it?

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by adizere** [comment, marked as answer]
Hi,

In general, we don't support a "restricted" mode that can do without gRPC connectivity.

But it depends on your use-case. What would you like to achieve exactly? If you're looking to simply do some queries about the IBC state of Akash Testnet-3 (e.g., query a client state might work without gRPC), then you can try to set `grpc_addr` to some incorrect address and its possible Hermes will work, though note we never tested this scenario. If you want to do any more advance query or submit transactions to/from Akash Testnet-3, then gRPC is necessary.

**Reply by simpletrontdip** [reply]
Thanks,

I got a `transport error` for setting it to some random value.
I'm creating an IBC channel and may submit transactions, so seems `grpc` is mandatory here.
I will try to address this on Akash's side then.
